SU2_CFD/include/output/filewriter/CSurfaceFVMDataSorter.hpp

```diff
@@ -67,13 +67,40 @@ class CSurfaceFVMDataSorter final: public CParallelDataSorter{
 
   /*!
    * \brief Get the global index of a point.
-   * \input iPoint - the point ID.
+   * \param[in] iPoint - the local renumbered point ID.
    * \return Global index of a specific point.
    */
-  unsigned long GetGlobalIndex(unsigned long iPoint)  override{
+  unsigned long GetGlobalIndex(unsigned long iPoint) override{
```

Comment:
This took me a bit because this routine `GetGlobalIndex `returns a value of a (class data) std::map<ul, ul> via operator[] ... which is not const [stackoverflow](https://stackoverflow.com/questions/1474894/why-isnt-the-operator-const-for-stl-maps) . Using the `.at(...)` function was the way out ([compilerExplorer code](https://gcc.godbolt.org/z/esFpGx))